components/web_discovery/browser/pref_names.h

```diff
@@ -20,6 +20,14 @@ inline constexpr char kCredentialRSAPrivateKey[] =
 inline constexpr char kAnonymousCredentialsDict[] =
     "brave.web_discovery.anon_creds";
 
+inline constexpr char kScheduledDoubleFetches[] =
+    "brave.web_discovery.scheduled_double_fetches";
+inline constexpr char kScheduledReports[] =
+    "brave.web_discovery.scheduled_reports";
```

Comment:
I think to avoid possible bugs, this should be something like:

```
using QueuePrefNames = base::StrongAlias<class NameTag, std::string_view>;
inline constexpr auto kScheduledDoubleFetches = QueuePrefNames("src");
inline constexpr auto kScheduledReports = QueuePrefNames("referrerpolicy");
```

This way we can pass them to the constructor of `RequestQueue` knowing they are not just some dangling reference.

Reply:
Updated comment above.